powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java

```diff
@@ -861,14 +858,14 @@ public LookupApplicationByAppKeyResponse lookupApplicationByAppKey(LookupApplica
     @Override
     @Transactional
     public CreateApplicationResponse createApplication(CreateApplicationRequest request) throws GenericServiceException {
-        if (request.getApplicationName() == null) {
+        if (request.getApplicationId() == null) {
             logger.warn("Invalid request parameter applicationName in method createApplication");
```

Comment:
`applicationName` survived in the log message?

Reply:
Good catch